config/default.yaml

```diff
@@ -19,11 +19,11 @@ pcipalantenna:
 s2s:
   url: http://localhost:23443
 payhub:
-  url: http://localhost:8081
+  url: http://payment-api-demo.service.core-compute-demo.internal
 bulkscan:
   url: http://localhost:7070
 refunds:
-  url: http://localhost:8084
+  url: http://ccpay-refunds-api-demo.service.core-compute-demo.internal
 notification:
```

Comment:
We don't want http://ccpay-refunds-api-demo.service.core-compute-demo.internal and any other demo mention to be merge it on master.